purchase_propagate_qty/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+* Matthieu Mequignon <matthieu.mequignon@camptocamp.com>
```

Comment:
I think we should see original authors too here

Reply:
Well, I just appended my name there…
https://github.com/OCA/purchase-workflow/blob/12.0/purchase_propagate_qty/README.rst